visual-js/visual-snapshots/src/api/visual-snapshots-api.ts

```diff
@@ -14,32 +13,26 @@ export interface CreateVisualSnapshotsParams {
   snapshotName?: string;
 }
 
-export class VisualSnapshotsApi {
-  constructor(
-    private readonly api: VisualApi,
-    private readonly pdfSnapshotUploader: PdfSnapshotUploader
-  ) {}
-
-  public async generateAndSendPdfFileSnapshots(
-    pdfFilePaths: string[],
-    params: CreateVisualSnapshotsParams
-  ) {
-    const buildId = params.buildId ?? (await this.createBuild(params));
+export interface CreateBuildParams {
```

Comment:
I like splitting `VisualSnapshotsApi` into `CreateBuildParams` and `UploadSnapshotParams`  :)